agent_code/our_agent_v2/train.py

```diff
@@ -271,7 +305,9 @@ def end_of_round(self, last_game_state: dict, last_action: str, events: List[str
 
         #inkpca.partial_fit()
 
-
+    # ---------- (5) Clean n step transition history: don't compute aggregated reward beyond one game ----------
+    
+    self.n_step_transitions = deque([], N_STEPS) 
```

Comment:
`self.n_step_transitions.clear()` empties the que

Reply:
```suggestion
    self.n_step_transitions.clear()
```